cmd/tempo-vulture/main.go

```diff
@@ -65,17 +65,16 @@ func main() {
 
 	// Write
 	go func() {
+		c, err := newJaegerGRPCClient(tempoPushURL)
+		if err != nil {
+			glog.Error("error creating grpc client", err)
+			metricErrorTotal.Inc()
```

Comment:
Well, if this happens, Vulture won't work... Idk if this is the best way to handle this...

Actually, It's going to crash.

Reply:
For example, if the tempoPushURL is malformed.

Reply:
Replaced this with panic...